eng/pipelines/coreclr/templates/build-perf-bdn-app.yml

```diff
@@ -81,7 +81,17 @@ steps:
     displayName:  Clone Maui
     workingDirectory: $(Build.SourcesDirectory)
 
+  - script: |
+      set -x
+      pwd
+      git clone https://github.com/dotnet/performance.git --depth 1 -b ${{parameters.perfRepo}} --single-branch
+    displayName:  Clone performance
+    workingDirectory: $(Build.SourcesDirectory)
+
   - pwsh: |
+      $BenchmarkDotNetVersionCapture = Get-Content .\performance\eng\Versions.props | Select-String -Pattern '<BenchmarkDotNetVersion>(.*)</BenchmarkDotNetVersion>'
+      $BenchmarkDotNetVersion = $BenchmarkDotNetVersionCapture.Matches.Groups[1].Value
```

Comment:
Check here also that the match was successful?

Reply:
Added a check for this.

Reply:
Please take a quick look at this match check an let me know if you have any thoughts on better ways to do the check. Thanks!

Reply:
I don't have a strong opinion here. `$BenchmarkDotNetVersionCapture.Matches[0].Success` or `...Matches.Groups[1].Success`  might be other option.